<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntellectVideoViewClass</class>
 <widget class="QWidget" name="IntellectVideoViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1570</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HomeView</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="7,3">
   <item>
    <widget class="QWidget" name="widget_8" native="true">
     <property name="Style_Widget" stdset="0">
      <string>Style_White_Widget</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QWidget" name="widget_11" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget[Style_Widget=&quot;Video&quot;]
{

border-radius:10px;
background-color: rgb(232, 236, 240);
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <widget class="VideoView" name="widget" native="true">
           <property name="Style_Widget" stdset="0">
            <string>Video</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_13" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QAbstractButton {
	padding-top: 10px;
    border-radius: 5px;
    border: 2px solid #525252;
    background-color: #FFFFFF;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="PixmapWidget" name="btnVideoCall" native="true">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>130</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PixmapWidget" name="btnAudioCall" native="true">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>130</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PixmapWidget" name="btnCallOver" native="true">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>130</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PixmapWidget" name="btnUp" native="true">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>130</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PixmapWidget" name="btnStatusChange" native="true">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>130</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PixmapWidget" name="btnStop" native="true">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>130</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PixmapWidget" name="btnDown" native="true">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>130</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PixmapWidget" name="btnAuto" native="true">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>130</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_9" native="true">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="3,2,2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="Style_Widget" stdset="0">
         <string>Style_White_Widget</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>播放列表</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_16" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QListWidget" name="listWidgetPlayer"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="Style_Widget" stdset="0">
         <string>Style_White_Widget</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_15" native="true">
        <property name="Style_Widget" stdset="0">
         <string>Style_White_Widget</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>文件夹</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btnDelete">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>删除</string>
           </property>
           <property name="icon">
            <iconset resource="../../../resources/QtIntellectNurse.qrc">
             <normaloff>:/image/common/删除.png</normaloff>:/image/common/删除.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btnOpen">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>打开</string>
           </property>
           <property name="icon">
            <iconset resource="../../../resources/QtIntellectNurse.qrc">
             <normaloff>:/image/common/打开.png</normaloff>:/image/common/打开.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QWidget" name="widget_19" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QListWidget" name="listWidget"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PixmapWidget</class>
   <extends>QWidget</extends>
   <header>Component/PixmapWidget/PixmapWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VideoView</class>
   <extends>QWidget</extends>
   <header>Component/VideoView/VideoView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources/QtIntellectNurse.qrc"/>
 </resources>
 <connections/>
</ui>
